gmaps/directions.py

```diff
@@ -133,6 +151,23 @@ def directions_layer(start, end, waypoints=None):
         Google maps imposes a limitation on the total number of waypoints.
         This limit is currently 23.
     :type waypoints: List of 2-element tuples, optional
+
+    :param avoid_ferries If true, instructs the Distance Matrix service to avoid 
```

Comment:
I think adding which GoogleMaps service is affected by parameters will confuse most users -- how about just "avoid ferries where possible"?

Also, you're missing a colon after the parameter name:

```
:param avoid_ferries: avoid ferries where possible (default: False)
```

(the same is true for the other parameters here)